### ISSUE TITLE
Use latex instead of mathjax for Anki cards

### ISSUE THREAD

**Issue by WinterDawn** [eyeinsky/org-anki#86]
I’m using some packages not supported by mathjax. Is there a way I can export to Anki cards without converting latex to mathjax?

**Comment by eyeinsky** [comment]
Not currently, but I can look into adding that. Would leaving the latex as is, work?

**Comment by WinterDawn** [comment]
Yes, just leave it unchanged and surrounded by [latex] tag would work.
